website/sidebarsApi.js

```diff
@@ -173,6 +173,11 @@ export default {
       label: "Reference",
       collapsed: true,
       items: [
+        {
+          type: "doc",
+          label: "Recursion",
+          id: "recursion",
+        },
```

Comment:
locally on the sidebar, I do not see this item at all (maybe an error from link issues?) please confirm

Reply:
so far I'm only working in `/api/` and `sidebarsApi.js`, which means the changes only appear on `next`.

will duplicate to `/api_versioned_sidebars/version-1.0` and `api_versioned_sidebars/sidebar-version-1.0` once all the copy editing and content review are figured out.